velox/docs/functions/iceberg/bucket.rst

```diff
@@ -0,0 +1,8 @@
+=============================
+Bucket Function
+=============================
+.. iceberg:function:: bucket(numBuckets, input) -> integer
+
+   Returns an integer between 0 and ``numBuckets - 1`` representing the bucket assignment. ::
```

Comment:
What is "bucket"? How is it computed? What types are supported for numBuckets and input? Can you provide some more context?